lib/funky/page.rb

```diff
@@ -12,7 +12,11 @@ class Page < GraphRootNode
     # @return [Funky::Page] containing the data fetched by Facebook Graph API.
     def self.find(page_id)
       page = Funky::Connection::API.fetch("#{page_id}?fields=name,username,location,fan_count,featured_video")
-      new(page)
+      if page[:name].nil?
```

Comment:
"page" does not have `type` field. But there's documentation for "url" https://developers.facebook.com/docs/graph-api/reference/v2.8/url (and it has `type` field under `og_object`)